opflexagent/utils/port_managers/async_port_manager.py

```diff
@@ -112,7 +112,7 @@ def schedule_update(self, device_ids=None):
         device_ids = set(device_ids or [])
         LOG.debug('Update initially scheduled for port ids %s', device_ids)
         # See if more ports need to be updated due to request timeout
-        for request in self.pending_requests.get_requests():
+        for request in list(self.pending_requests.get_requests()):
```

Comment:
Is this list() still needed?

Reply:
Yes. Without wrapping of list, it gives: "RuntimeError: dictionary changed size during iteration"

Reply:
@kentwu1 If you agree with the code, can you approve the changes? Thanks!